docs/levenscyclus/index.md

```diff
@@ -1,32 +1,37 @@
 --- 
 title: Levenscyclus
+summary: Hulp per fase van de levenscyclus van algoritmes en AI, zoals probleemanalyse, ontwerpen, ontwikkelen, implementeren en ermee stoppen.
 icon: material/reload
 --- 
+# Levenscyclus algoritmes en AI
 
-!!! info "Disclaimer"
+Verantwoordelijke algoritmes en AI voldoen tijdens elke fase van de levenscyclus aan de regels. Van het nemen van een besluit tot het stoppen met algoritmes of AI.
```

Comment:
```suggestion
Verantwoord gebruik van algoritmes en AI betekent dat je moet tijdens elke fase van de levenscyclus aan de regels moet voldoen. Van het nemen van een besluit tot het stoppen met algoritmes of AI.
```

Reply:
verantwoordelijke algoritmes bestaan volgens mij niet. Het gaat om het verantwoord gebruik of inzet van de algoritmes

Reply:
ook hier twijfels over het woord 'regels'

Reply:
```suggestion
Bij verantwoord gebruik van algoritmes en AI voldoe je tijdens elke fase van de levenscyclus aan de juiste regels. Van het nemen van een besluit tot het stoppen met algoritmes of AI.
```

Reply:
Heb je suggestie iets herschreven omdat het op deze pagina niet gaat om de betekenis van 'verantwoord gebruik van AI', maar om de relevantie van de levenscyclus voor de gebruiker. Kun je je er zo in vinden?

Reply:
zie comment hieronder